src/main/java/com/useful/ucars/uCarsListener.java

```diff
@@ -745,9 +745,8 @@ public void onUcarUpdate(ucarUpdateEvent event) {
 			}
 			UEntityMeta.removeMetadata(car, "car.jumping");
 		}
-		if(car instanceof Minecart) {
-			((Minecart)car).setMaxSpeed(5); // Don't allow game breaking speed - but faster
-			// than default
+		if(car instanceof Minecart && ((Minecart) car).getMaxSpeed() != 5) {
+			((Minecart)car).setMaxSpeed(5); // Don't allow game breaking speed - but faster than default
```

Comment:
Was this having some performance implication? Not sure what prompted this change

Reply:
Hmm I've never seen it show up in any timings

Reply:
Yeah seems within spigot it is just directly writing to a single field, so should take basically no time whatsoever

Reply:
I can change it back if you want